azure-kusto-data/azure/kusto/data/__init__.py

```diff
@@ -1,5 +1,242 @@
-from .kusto_client import KustoClient, KustoResultIter, KustoResponse
-from .kusto_exceptions import KustoError, KustoClientError, KustoServiceError
+import uuid
+import requests
+from enum import Enum, unique
+
+from six import text_type
+
+from ._response import _KustoResponse
+from .exceptions import KustoServiceError as _KustoServiceError
+from .security import _AadHelper
 from .version import VERSION
 
 __version__ = VERSION
+
+
+class KustoConnectionStringBuilder(object):
+    """Parses Kusto conenction strings.
+
+    For usages, check out the sample at:
+        https://github.com/Azure/azure-kusto-python/blob/master/azure-kusto-data/tests/sample.py
+    """
+
+    @unique
+    class _ValidKeywords(Enum):
+        data_source = "Data Source"
+        aad_user_id = "AAD User ID"
+        password = "Password"
+        application_client_id = "Application Client Id"
+        application_key = "Application Key"
+        authority_id = "Authority Id"
+
+    _Keywords = {
```

Comment:
Valid keywords are in Src\Client\Kusto.Data\DataProvider\KustoConnectionStringBuilder.cs
I didn't see Addr, Address, NEtwork Address there